### ISSUE TITLE
Downloading multiple torrents

### ISSUE THREAD

**Issue by benjaminrobinet** [mafintosh/torrent-stream#147]
Could you please make an update to be able to download multiple torrents like WebTorrent ?

**Comment by bsuh** [comment]
There's nothing wrong with calling `torrentStream("magnet-link")` multiple times.

**Comment by benjaminrobinet** [comment]
yes but the needed way should be engine.add("torrent"). Like webtorrent

**Comment by HazCod** [comment]
Are there any technical problems regarding having around 300-500 torrents running?
Will there be 500+ connections open? And what about peerlists?

**Comment by iadj** [comment]
Also interested in this. Is it viable to use torrent stream for managing (down- and uploading) a large amount of torrents? i.e. use it as the core torrent component for a base torrent manager?

**Comment by asapach** [comment]
I'm using it in https://github.com/asapach/peerflix-server with multiple instances sharing the same DHT

**Comment by iadj** [comment]
Good to hear. Thanks @asapach!
